core/src/main/scala/caliban/introspection/adt/__Field.scala

```diff
@@ -12,7 +12,8 @@ case class __Field(
   `type`: () => __Type,
   isDeprecated: Boolean = false,
   deprecationReason: Option[String] = None,
-  @GQLExcluded directives: Option[List[Directive]] = None
+  @GQLExcluded directives: Option[List[Directive]] = None,
+  @GQLExcluded tags: Set[String] = Set.empty
```

Comment:
I'm not crazy happy about adding another argument to the case class here, but I tried to use directives for it but they ended up showing in the rendered schema. Happy to consider alternatives if we don't like this

Reply:
Actually that's one thing I was not happy with in that blog post: https://blog.pierre-ricadat.com/graphql-in-scala-role-based-access-control: the directives show up in the schema and you might not want that. In that particular example I could now use those tags instead of the directives, but I am wondering if just String is powerful enough (you might want more, for example the roles were typed in my example). How about using directives but making it configurable whether they appear in the introspection/rendering?

Reply:
I see your point. I think adding a way to hide directives from rendering/introspection makes sense.

However,how about we control it at the directive definition instead? Perhaps by adding a boolean argument `isIntrospectable` to the class that defaults to `false`?

Reply:
Yeah I think that makes sense.

Reply:
> How about using directives but making it configurable

Oh wait, this is what you recommended here. For some reason I thought you recommended to add some sort of config that can be passed to introspection / rendering

Reply:
Yeah I think that makes sense = Yeah I think my idea makes sense 🤣